### DISCUSSION TITLE
ovos stopped speaking after install "ovos-stt-plugin-chromium"

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#246] [Q&A] [answered]
followed doc'd steps.   ovos no longer speaks  right after installed. "ovos-stt-plugin-chromium"   no unit is dead.   how restore to working state?

**Comment by mushie4282** [comment]
I have the same issue! In my experience, `25Feb2024_2137_raspOVOS-headless-dev.zip`
 and `26Feb2024_0647_raspOVOS-headless-dev.zip` don't have STT working properly. `ovos-stt-plugin-chronium` doesn't work on either images. I have not tested out `02Mar2024_2137_raspOVOS-headless-dev.zip` yet.

**Comment by geetee24** [comment]
Do you have any advice on how to get the original SST to work which is
basically useless cause it takes five seconds to answer any question? So
frustrating.

**Comment by mikejgray** [comment]
What do the logs say? I believe in RaspOVOS it is /ramdisk/logs and you'll be looking for a filename that says listener or speech or voice. Even with issues in the image, they can usually be corrected without re-flashing.

**Reply by mushie4282** [reply]
None of the log files mention STT loading. Here are the last logs of audio.log.
`2024-03-08 17:45:06.800 - audio - ovos_audio.service:run:243 - WARNING - audio service has moved to ovos-media, if you already migrated to ovos-media set "enable_old_audioservice": false in mycroft.conf
2024-03-08 17:45:06.903 - audio - ovos_audio.service:run:247 - WARNING - No audio backends loaded! Audio playback is not available
2024-03-08 17:45:06.904 - audio - ovos_audio.service:run:249 - INFO - Running audio service in TTS only mode`

**Reply by JarbasAl** [reply]
you want the speech/listener log,  audio log is for audio output (TTS) not input (STT)

**Comment by JarbasAl** [comment]
> followed doc'd steps. ovos no longer speaks right after installed. "ovos-stt-plugin-chromium" no unit is dead. how restore to working state?

please share error logs otherwise we are just guessing what happened

**Comment by geetee24** [comment]
referenced: https://github.com/OpenVoiceOS/ovos-stt-plugin-chromium

1. installed pip install ovos-stt-plugin-chromium
2. edited: sudo nano ../.config/mycroft/mycroft.conf
3. contents:
{
  "log_level": "DEBUG",
  "system_unit": "imperial",
  "websocket": {
    "host": "0.0.0.0"
  },
  "stt": {
    "module": "ovos-stt-plugin-chromium",
    "ovos-stt-plugin-chromium": {
        "lang": "en-US",
        "pfilter": false,
        "debug": false
    }
  }
}

OVOS wont respond to my questions due to:
voice log:
2024-03-09 09:17:28.868 - voice - ovos_dinkum_listener.plugins:finalize:35 - ERROR - Error in STT plugin: ChromiumSTT
Traceback (most recent call last):
  File "/home/ovos/.venv/lib/python3.11/site-packages/ovos_dinkum_listener/plugins.py", line 30, in finalize
    transcript = self.engine.execute(audio, self.language)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ovos/.venv/lib/python3.11/site-packages/ovos_stt_plugin_chromium/__init__.py", line 221, in execute
    flac_data = audio.get_flac_data(
                ^^^^^^^^^^^^^^^^^^^^
  File "/home/ovos/.venv/lib/python3.11/site-packages/speech_recognition/audio.py", line 232, in get_flac_data
    flac_converter = get_flac_converter()
                     ^^^^^^^^^^^^^^^^^^^^
  File "/home/ovos/.venv/lib/python3.11/site-packages/speech_recognition/audio.py", line 290, in get_flac_converter
    raise OSError(
OSError: FLAC conversion utility not available - consider installing the FLAC command line application by running `apt-get install flac` or your operating system's equivalent
2024-03-09 09:17:28.872 - voice - ovos_dinkum_listener.voice_loop.voice_loop:_after_cmd:680 - INFO - nothing transcribed


I installed "apt-get install flac"  and restarted yet ovos STILL DEAD

**Reply by JarbasAl** [reply]
> OSError: FLAC conversion utility not available - consider installing the FLAC command line application by running apt-get install flac or your operating system's equivalent

whats the next error after installing flac?

**Comment by geetee24** [comment]
to get out of bricked state, what line entry should I add in for fallback -?  What is the default STT name?

**Reply by JarbasAl** [reply]
remove your changes to the mycroft.conf and it will use default values again

**Reply by geetee24** [reply]
it does not recover when I remove my changes.   rebooting IT NEVER SPEAKS again.

**Reply by JarbasAl** [reply, marked as answer]
well, that sucks, but without errors logs it's impossible to help

**Comment by geetee24** [comment]
already gave you all the logs.  nothing new in the logs

**Reply by JarbasAl** [reply]
seems i cant help you then :shrug: try turning it on and off again

you gave me logs that clearly show you using chromium plugin, the solution for that error was in the logs themselves and was given to you here again, you never said if that worked but asked how to revert, i told you to revert your changes in mycroft.conf to go back to the defaults, i can't guess what happened next or what your actions were
